accounting/invoice/models.py

```diff
@@ -332,7 +299,13 @@ def save(self, *args, **kwargs):
         if self.auto_create_pdf_on_save:
             self.to_pdf()
         if self.auto_upload_google_drive_on_save:
-            self.upload_to_google_drive()
+            file = self.upload_to_google_drive(
+                file_path=self.pdf_path,
+                target_path=[self.date.strftime('%Y'), 'invoices-in', self.date.strftime('%m')],
+                title=self.pdf_filename,
+            )
+            self.pdf_path = file['webContentLink']
+            Invoice.objects.filter(pk=self.pk).update(pdf_path=self.pdf_path)
```

Comment:
Actually, it's not just to make it show up as a download link, but to properly set the variable, which is more important. If we don't do this, the variable won't be set because we already call the super's save above, and we can't call it again lest infinite recursion.

I will document this behavior.